task-manager/src/test/java/ua/knu/knudev/query/dsl/QueryDslTest.java

```diff
@@ -0,0 +1,137 @@
+package ua.knu.knudev.query.dsl;
```

Comment:
In Telegram we disscussed problems with this tests, will fix in future

Reply:
ok